helios-client/src/test/java/com/spotify/helios/common/JobValidatorTest.java

```diff
@@ -416,4 +417,31 @@ public void testExpiry() {
         .setExpires(d).build();
     assertEquals(newHashSet("Job expires in the past"), validator.validate(j));
   }
+
+  @Test
+  public void testWhitelistedCapabilities_noWhitelist() throws Exception {
+    final Job job = Job.newBuilder()
+        .setName("foo")
+        .setVersion("1")
+        .setImage("foobar")
+        .setAddedCapabilities(ImmutableSet.of("cap1", "cap2"))
+        .build();
+
+    assertEquals(1, validator.validate(job).size());
```

Comment:
to be extra careful with the set math, might want to test which capability was deemed to be not allowed